app/views/categories/show.html.slim

```diff
@@ -0,0 +1,10 @@
+h2
+  = @category.title
+h4
+  = t('budget')
```

Comment:
Maybe it worth to move this locals to:
```
en: 
  categories:
    show:
      budget: ''
```
rather than using plain yml file?
You can use `t('.budget')` for smart scoping for locals according to file.